ST_mortality.c

```diff
@@ -268,106 +278,62 @@ void (*simulatePrescribedFire)(GrpIndex, RealF) = _simulatePrescribedFire;
  * \ingroup MORTALITY
  */
 void mort_EndOfYear(void) {
-    GrpIndex rg;
-    GroupType *g = NULL;
-    SppIndex sp;
-    IntU j;
-    RealF fire_possibility, random_number, biomass_cheatgrass;
-    char *cheatgrass_name = "brte";
-    int i = 0;
-    Bool prescribed_fire_on = FALSE;
-
-    /* Check species index number from the beginning to all the species in
-     *  species.in , if the species name == checkname then get the biomass and stop the loop*/
-    for (i = 0; i < Globals->sppCount; i++) { /* if species name = checkname = brte then get the biomass of brte(cheatgrass)*/
-        if (strcmp(cheatgrass_name, Species[i]->name) == 0) {
-            biomass_cheatgrass = Species_GetBiomass(i); /* calculate biomass of cheatgrass*/
-            g = RGroup[Species[i]->res_grp];
-            break;
-        }
-    }
+  GrpIndex rg;
+  SppIndex sp;
+  IntU j;
+  RealF cheatgrassBiomass;
+
+  /* Update the precipitation parameters that determine cheatgrass-driven
+   * wildfire. */
+  _updateCheatgrassPrecip(Globals->currYear);
+  /* printf("%d:\tMeanSpring = %f\t ThisSpring = %f\t MeanWinter = %f\t LastWinter = %f\n",
```

Comment:
@chaukap - since we are not currently using this function , I believe we should comment this function for the time being and add a comment explaining that it can be incorporated at a later time. It is slowly down the program otherwise.

Reply:
Resolved by commit bae2906.